webapp/app/js/directives/persona.js

```diff
@@ -19,6 +19,8 @@ treeherder.directive('personaButtons', [
                 // if the user.email value is null, it means that he's not logged in
                 $rootScope.user.email = response.data.userEmail || null;
                 $rootScope.user.loggedin = $rootScope.user.email !== null;
+                $rootScope.user.is_staff= true;
```

Comment:
What's this for?  Was the "only show the cancel button on non-try if is_staff" feature broken before?